### DISCUSSION TITLE
How do you use a pack in offline mode?

### DISCUSSION THREAD

**Discussion by devWaleed** [rnmapbox/maps#2336] [Q&A] [answered]
On app start, on app mount I download the pack on my device and in progress listeners it confirms me that pack is downloaded.

`
 MapboxGL.offlineManager.createPack(
        {
          name: packName,
          styleURL: styleURL,
          minZoom: 20,
          maxZoom: 20,
          bounds: [
            [67.203728, 24.94847],
            [66.814057, 24.846638],
          ],
        },
        progressListener,
        errorListener,
      );
`

Now, I am unable to find a way to use this downloaded map in offline map. Docs are not very clear about this.

I am on version     "@rnmapbox/maps": "^10.0.0-beta.48",

**Comment by mfazekas** [comment, marked as answer]
Stye pack should be consumed automatically. So if you createPack and download an area, then you render a map using that area it should render the map in offline mode.

https://docs.mapbox.com/ios/maps/guides/offline/
